api/src/pcapi/core/offers/api.py

```diff
@@ -244,6 +244,7 @@ def create_draft_offer(
         isActive=False,
         validation=models.OfferValidationStatus.DRAFT,
         product=product,
+        ean=fields.get("extraData", {}).get("ean", None),
```

Comment:
Je ferai ça plutôt comme plus haut, en utilisant `fields.update()`
Quelque chose comme
```suggestion
        ean = body.extra_data.get("ean", None) # à adapter pour le cas où il n'y a pas d'ExtraData
        # ...
        fields.update("ean", ean)
```